app/interactors/validate_age.rb

```diff
@@ -4,8 +4,7 @@ class ValidateAge
   include Interactor
 
   def call
-    context.fail!(error: I18n.t('too_young')) if context.age < 18
-
-    context.fail!(error: I18n.t('too_old')) if context.age > 69
+    context.fail!(error: I18n.t('too_young')) if context.quote.age < 18
```

Comment:
Whoops! This interactor should have been using age from the quote since I moved quote creation to the start of the chain.